backend/de.metas.util/src/test/java/de/metas/util/time/SystemTimeTest.java

```diff
@@ -80,5 +84,23 @@ public void test_asZonedDateTimeAtStartOfDay()
 		assertThat(dayTS.getSecond()).isEqualTo(0);
 		assertThat(dayTS.getNano()).isEqualTo(0);
 	}
+	
+	@Test
+	public void test_2359()
```

Comment:
what's the purpose of this test? (i.e. what it tests)
what's the relation with SystemTime class?

Reply:
As per my concept, I wanted to make sure that particular line
```java
final ZonedDateTime todayEndOfDay = SystemTime.asLocalDate().atTime(LocalTime.MAX).atZone(SystemTime.zoneId());
```
is retrieving the date with hour 23:59. 
True, it doesn't actually test SystemTime. I'll just take it out. It's not such a relevant test. I suppose LocalTime.MAX won't change over time.